server/broadcast.go

```diff
@@ -32,17 +32,13 @@ func (pi *packetInfo) broadcastMessage() error {
 		if vk == broadcastKey {
 			err := pi.broadcastAll(msgs)
 			if err != nil {
-				pi.broadcastNewRound(true)
-
-				// Don't disconnect, we broadcasted a new round.
+				// Don't disconnect
```

Comment:
I think we can remove the error return value. We don't return any errors from this method and we can probably change the function signature.

Reply:
Probably can change in a follow up PR. =)

Reply:
Will do.